README.md

```diff
@@ -24,9 +24,9 @@ Setup:
    - Twitter: https://twitter.com/custodiolanga1
    - LinkedIn: https://www.linkedin.com/in/custodio-serafim-2a318a23a 

```

Comment:
- [ ] Please kindly update the `Author's information` for `Author 1 and Author 2` to match that of `Author 3` as this will make your readme file look for professional. Please also make sure the Author's information have links to the various platforms(GitHub, LinkedIn, Twitter). 
- [ ] It will be nice to update your `Getting started section` with explicit steps on how to clone and run your app locally as this will be of help to non technical users. 
- [ ] Please kindly update your `License section` by providing a link to the license. Checkout this [template](https://github.com/microverseinc/readme-template) for clarifications.